src/bp/ui-studio/src/web/components/Layout/Sidebar.tsx

```diff
@@ -17,20 +17,30 @@ type Props = StateProps & RouteComponentProps
 
 const BASIC_MENU_ITEMS = [
   {
+    id: 'content',
     name: lang.tr('content'),
     path: '/content',
     rule: { res: 'bot.content', op: 'read' },
     icon: 'description'
   },
   {
+    id: 'flows',
     name: lang.tr('flows'),
     path: '/flows',
     rule: { res: 'bot.flows', op: 'read' },
     icon: 'page-layout'
+  },
+  {
+    id: 'nlu',
+    name: lang.tr('studio.sideBar.nlu'),
+    path: '/nlu',
```

Comment:
this is how you inject the nlu views right ?

Reply:
There is no injection this is a simple history push